generator/template_repo.go

```diff
@@ -207,6 +208,15 @@ func asPrettyJSON(data interface{}) (string, error) {
 	return string(b), nil
 }
 
+func asGoInitializer(data interface{}) (string, error) {
+	// Constructs a Go literal initializer from interface{}
+	b, err := json.Marshal(data)
+	if err != nil {
+		return "", err
+	}
+	return strings.Replace(strings.Replace(strings.Replace(string(b), "}", ",}", -1), "[", "{", -1), "]", ",}", -1), nil
```

Comment:
bytes has the same set of methods, so you don't strictly need to convert to string to do this.
How does this deal with property names?

Reply:
This part is under full rework as it is not satisfactory and did not pass my internal round of testing.

Initialization of default values for parameters is actually quite a bit more complex than expected,
when considering formats (which do not expose a consistent initialization interface), arrays and nested arrays.

This method was originally intended to initialize slices.

We'll have another round of discussion on this with forthcoming commit

Reply:
I am not sure that `string(bytes.Replace(.... Replace(b,[]byte("}"), [}byte(",}",-1), []byte("["), []byte("{"....)`
would be more readable... 
I am adding tests to demonstrate use with properties, even if the initial intent is especially to deal with slices.